idseq_pipeline/commands/host_filtering_functions.py

```diff
@@ -564,6 +567,11 @@ def run_host_filtering(fastq_files, initial_file_type_for_log, lazy_run, stats):
                       after_filename=os.path.join(RESULT_DIR, EXTRACT_UNMAPPED_FROM_SAM_OUT1),
                       after_filetype="fasta_paired")
 
+def upload_pipeline_version_file():
+    execute_command("echo %s > %s" % (PIPELINE_VERSION, PIPELINE_VERSION_OUT))
+    execute_command("aws s3 cp %s %s/" % (PIPELINE_VERSION_OUT, OUTPUT_BUCKET))
```

Comment:
If OUTPUT_BUCKET does not end in "/" and has not yet been created, the result here is likely not the one intended.

Reply:
why? see '/' in the command

Reply:
i missed that, thanks